src/pages/policies/policy-list.tsx

```diff
@@ -23,7 +23,7 @@ export default function PolicyList(props: Props) {
         <li key={policy.policyId} className={styles.policyItem}>
           <div className={styles.policyItemMain}>
             <Link className={styles.policyItemTitle} to={`/policies/${policy.policyId}`}>
-              Policy {policy.ipfsHash}
+              {policy.metadata}
```

Comment:
Will there be some guarantee on metadata length? (Just asking whether because a long string might break/not-look-good in the UI)

Reply:
There is already CSS in place that truncates the text when it gets too long (like we do for proposals) 👍

Reply:
Perfect. Thanks.